doc/related_projects.rst

```diff
@@ -207,9 +168,6 @@ and tasks.
   <https://en.wikipedia.org/wiki/Variational_Bayesian_methods>`_ to sample from
   a tractable approximation of a topic model's posterior distribution.)
 
-- `Sparse Filtering <https://github.com/jmetzen/sparse-filtering>`_
```

Comment:
Minimal package, no changed since 2014. We could consider adding it to `scikit-learn-extra`